kong/llm/plugin/shared-filters/normalize-json-response.lua

```diff
@@ -38,6 +38,22 @@ local function transform_body(conf)
     response_body = cjson.encode({ error = { message = err }})
   end
 
+  local t, err
```

Comment:
```suggestion
  -- TODO: avoid json encode and decode when transforming
  --             deduplicate body usage parsing from parse-json-response
  local t, err
```
leave this as TODO, not a blocker.